pkg/controller/common/controller_context.go

```diff
@@ -54,6 +58,16 @@ func CreateControllerContext(cb *clients.Builder, stop <-chan struct{}, targetNa
 	sharedNamespacedInformers := mcfginformers.NewFilteredSharedInformerFactory(client, resyncPeriod()(), targetNamespace, nil)
 	kubeSharedInformer := informers.NewSharedInformerFactory(kubeClient, resyncPeriod()())
 	kubeNamespacedSharedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), targetNamespace, nil)
+	// This informer is specifically filtering for mco-built-in labels for cluster-role and cluster-role-bindings
+	filterClusterRoleAndClusterRoleBindings := func(opts *metav1.ListOptions) {
+		ls, err := labels.ConvertSelectorToLabelsMap(opts.LabelSelector)
+		if err != nil {
+			glog.Warningf("unable to convert selector %q to map: %v", opts.LabelSelector, err)
+			return
+		}
+		opts.LabelSelector = labels.Merge(ls, map[string]string{"mco-built-in/cluster-role": "", "mco-built-in/cluster-role-binding": ""}).String()
+	}
+	clusterRoleAndBindingsInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), targetNamespace, filterClusterRoleAndClusterRoleBindings)
```

Comment:
` NewSharedInformerFactoryWithOptions` instead of `NewFilteredSharedInformerFactory`